pages/jointhetribe.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import type { NextPage } from "next";
+import styles from "../styles/jointhetribe.module.css";
+import TextCopy from "../components/UI/textCopy";
+import Button from "../components/UI/button";
+import ClaimTribeModal from "../components/tribe/claimTribeModal";
+import { useAccount } from "@starknet-react/core";
+import FamousMembers from "../components/tribe/famousMembers";
+import { useDomainFromAddress } from "../hooks/naming";
+import SearchMembers from "../components/tribe/searchMembers";
+
+const JoinTheTribe: NextPage = () => {
+  const [isClaimModalOpen, setIsClaimModalOpen] = useState<boolean>(false);
+  const { address } = useAccount();
+  const mainDomain = useDomainFromAddress(address ?? "");
+
+  return (
+    <>
+      <div className={styles.page}>
+        <section className={styles.section1}>
+          <h1 className="title">Join the tribe</h1>
+          <div className={styles.centeredContainer}>
+            <p className="mr-5">Wear the stark signal</p>
+            <TextCopy text={mainDomain ? mainDomain : "yourname.stark"} />
+          </div>
+          <FamousMembers />
+        </section>
+
+        <div className={styles.firstLeaf}>
+          <img alt="leaf" src="/leaves/new/leaf_02.svg" />
+        </div>
+        <div className={styles.secondLeaf}>
+          <img alt="leaf" src="/leaves/new/leaf_03.svg" />
+        </div>
+        <div className={styles.thirdLeaf}>
+          <img alt="leaf" src="/leaves/new/leaf_02.svg" />
+        </div>
+        <div className={styles.fourthLeaf}>
+          <img alt="leaf" src="/leaves/new/leaf_03.svg" />
+        </div>
+        <section id="join" className={styles.section2}>
+          <h1 className="title">Get your tribe NFT !</h1>
+          <div className={styles.centeredContainer}>
+            <p className="text-center">
+              Join the tribe, to get the chance to have a special NFT !<br></br>
```

Comment:
"Join the tribe, to get the chance to receive a special NFT !" sounds better to me